polygeist/tools/cgeist/Test/Verification/move_alloca_to_top.cpp

```diff
@@ -0,0 +1,16 @@
+// RUN: mlir-translate -mlir-to-llvmir -split-input-file -verify-diagnostics %s | FileCheck %s
+
+// CHECK-LABEL: define void @move_alloca(float* %0) {
+// CHECK-DAG:     %{{.*}} = alloca float, i8 1, align 4
+// CHECK-DAG:     %{{.*}} = alloca float, i8 1, align 4
+// CHECK-NEXT:    %4 = load float, float* %0, align 4
```

Comment:
```suggestion
// CHECK-NEXT:    %{{.*}} = load float, float* %0, align 4
```

Reply:
After two alloca operation, the load must be %4.